modules/dnn/src/layers/fast_convolution/fast_convolution.cpp

```diff
@@ -0,0 +1,693 @@
+                                                                      ksize, inptr_ki += inp_planesize)
+                            {
+                                float t0 = inptr_ki[0], t1 = inptr_ki[stride_x];
+                                float t2 = inptr_ki[stride_x * 2], t3 = inptr_ki[stride_x * 3];
+                                inpbuf_ki[0] = t0;
+                                inpbuf_ki[1] = t1;
+                                inpbuf_ki[2] = t2;
+                                inpbuf_ki[3] = t3;
+                            }
+                        }
+                        i += 4;
+                        x0 += 4;
+                    }
+                    else
+                    {
+                        for (int c = 0; c < Cg; c++, inpbuf_ki += FAST_CONV_NR *
+                                                                  ksize, inptr_ki += inp_planesize)
+                            *inpbuf_ki = *inptr_ki;
+                        i++;
+                        x0++;
+                    }
+                }
+                else
+                {
+                    for (int c = 0; c < Cg; c++, inpbuf_ki += FAST_CONV_NR * ksize)
+                        inpbuf_ki[0] = 0.f;
+                    i++;
+                    x0++;
+                }
+                int mask = x0 >= W0;
+                y0 += mask;
+                x0 &= mask - 1;
+            }
+        }
+    }
+}
+
+static void doConvolution(float* outptr0, float* inpbuf_task, float* cbuf, const Ptr<FastConv2d>& conv, int HkWkCg,int k0, int k1, int yx0, int yx1,
+        size_t out_planesize, int g, int Kg, int Kg_aligned , bool partial0,
+        ActivationLayer*& activ, float minval, float maxval, bool ifMinMaxAct)
+{
+    int outstep0 = out_planesize;
+
+    for (int k = k0; k < k1; k += FAST_CONV_MR, outptr0 += outstep0 * FAST_CONV_MR)
+    {
+        int dk = Kg - k < FAST_CONV_MR ? Kg - k : FAST_CONV_MR;
+        bool partial = partial0 || dk < FAST_CONV_MR;
+        float *outptr = outptr0;
+
+        int outstep = outstep0;
+        if (partial)
+        {
+            outptr = cbuf;
+            outstep = FAST_CONV_NR;
+        }
+
+
+#if CV_TRY_AVX2
+        if (conv->useAVX2)
+            opt_AVX2::conv_block( HkWkCg, conv->weightsPtr + (g * Kg_aligned + k) * HkWkCg,
+                                  inpbuf_task, outptr, outstep, conv->biasPtr + Kg * g + k,
+                                  minval, maxval, ifMinMaxAct);
+        else
+#endif
+#if CV_TRY_AVX
+        if (conv->useAVX)
+            opt_AVX::conv_block( HkWkCg, conv->weightsPtr + (g * Kg_aligned + k) * HkWkCg,
+                                 inpbuf_task, outptr, outstep, conv->biasPtr + Kg * g + k,
+                                 minval, maxval, ifMinMaxAct);
+        else
+#endif
+#if CV_TRY_NEON
+        if (conv->useNEON)
+            conv_blockNEON(HkWkCg, conv->weightsPtr + (g * Kg_aligned + k) * HkWkCg,
```

Comment:
please insert _ before NEON for consistency

Reply:
fixed.